ccd-definition/CaseEventToFields/CreateSDO.json

```diff
@@ -24,6 +24,19 @@
     "CaseEventFieldLabel": " ",
     "FieldShowCondition": "drawDirectionsOrderRequired=\"Yes\""
   },
+  {
+    "CaseTypeID": "CIVIL",
+    "CaseEventID": "CREATE_SDO",
+    "CaseFieldID": "drawDirectionsOrderSmallClaims",
+    "PageFieldDisplayOrder": 1,
+    "DisplayContext": "MANDATORY",
+    "PageID": "ClaimsTrack",
+    "PageDisplayOrder": 2,
+    "PageColumnNumber": 1,
+    "ShowSummaryChangeOption": "Y",
+    "CaseEventFieldLabel": "Do you want to allocate this claim to the small claims track?",
```

Comment:
Is this the correct text, designs in ticket imply text should be "Is this claim suitable for allocation to the Small Claims Track?"

Reply:
Need to confirm during the demo as it differs in the micro board and Jira